suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -19,8 +19,10 @@ import { yup } from '@trezor/validation';
 
 import { Camera, CAMERA_HEIGHT } from '../components/Camera';
 
-const coinStyle = prepareNativeStyle(_ => ({
+const coinStyle = prepareNativeStyle(utils => ({
     flexDirection: 'row',
+    borderRadius: utils.borders.radii.large,
+    backgroundColor: 'rgba(255, 255, 255, 0.22)',
 }));
```

Comment:
Use colors from theme otherwise it will break when theme changes

Reply:
Fixed. I used transparentize instead to be able to have this modified when theme changes.